src/experimental-features.js

```diff
@@ -0,0 +1,26 @@
+export class ExperimentalFeatures {
+  constructor(storage) {
+    this.storage = storage;
+  }
+  getExperimentalFeatures() {
```

Comment:
What do you think about executing this and catching the value in the constructor?

I know that in that way if the local storage is updated we will not get the new values, but I think that in general, we do not want to do that.

On the other hand, reading the local storage is expensive and this code will be executed a lot of times, even when the feature is disabled (in production).

An alternative to read it in the constructor is using a lazy load pattern.